attractions/rollup.config.js

```diff
@@ -23,6 +24,20 @@ function prependTagOption(exceptions = []) {
   };
 }
 
+/**
+ * Recursively traverses all subdirectories of a given directory, listing all files it contains
+ * @param {string} dir Path to the directory whose files to list
+ * @returns {string[]} The list of files
+ */
+function getFiles(dir) {
```

Comment:
Maybe this is a better fit for a `utils` file?

Reply:
I mean, it's only useful here for the rollup task. Functions in the `utils` are suitable for the (web) app itself, which doesn't have node. Who would need to list files in a directory as part of their build process if not for a use case as specific as this one?